api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -193,6 +193,9 @@ def render_venue_details(
         dms_stats=dms_stats,
         delete_form=delete_form,
         active_tab=request.args.get("active_tab", "history"),
+        zendesk_sell_synchronisation_form=empty_forms.EmptyForm()
+        if utils.has_current_user_permission(perm_models.Permissions.MANAGE_PRO_ENTITY)
```

Comment:
On avait parlé de ne pas afficher le bouton si le lieu est virtuel ou non permanent, puisque ça échouera toujours. Je n'ai pas trouvé où tu mettais cette condition, si tu l'as bien oubliée je propose donc : 
```suggestion
        if venue.isPermanent and not venue.isVirtual and utils.has_current_user_permission(perm_models.Permissions.MANAGE_PRO_ENTITY)
```